JitsiConferenceEventManager.js

```diff
@@ -85,11 +74,14 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
     });
 
     chatRoom.addListener(XMPPEvents.AUDIO_MUTED_BY_FOCUS,
-        value => {
-            Statistics.analytics.sendEvent(REMOTELY_MUTED);
+        () => {
```

Comment:
This was a good catch. It's always a positive mute event and the promise below always sets isMutedByFocus true.